airflow/plugins_manager.py

```diff
@@ -41,6 +42,12 @@ class AirflowPlugin(object):
     admin_views = []
     flask_blueprints = []
     menu_links = []
+    # Defines which function to call per button
+    # key is the name of the button, which is a string
+    # value is the function to call to get the URL corresponding to the button
+    # The function should follow this format:
+    # type: (BaseOperator, datetime) -> str
```

Comment:
shouldn't type is (BaseOperator) -> str ?

Reply:
We also need the execution date. For example, we link you to logs for a task on a specific execution date in Kibana